examples/datasource-streaming-websocket/README.md

```diff
@@ -1,8 +1,14 @@
-# Grafana Streaming Data Source WebSocket Plugin example
+# Streaming WebSocket Data Source Plugin for Grafana
 
-This is an example of how to implement a Grafana data source plugin with streaming support.
+This repository provides an example of how to implement a Grafana data source plugin with streaming support.
 
-## Build
+## Overview
+
+The Streaming WebSocket Data Source Plugin illustrates the integration of WebSocket-based data sources into Grafana dashboards. This plugin serves as a reference implementation for developers aiming to incorporate real-time streaming data into their Grafana visualizations.
```

Comment:
We can't have two data sources claiming to be the same reference implementation... @oshirohugo is it time to just delete this one?

Reply:
This plugin examples do the streaming using a different approach than `datasource-streaming-backend-websocket`. It connects the frontend directly to a external websocket server. If connecting directly to a external server isn't something that we want plugin developers to do, than it should be removed. Otherwise it should be kept because this is not covered by `datasource-streaming-backend-websocket`.

Reply:
I just find it confusing as we also claim streaming as a backend capability https://grafana.com/developers/plugin-tools/introduction/backend-plugins#capabilities-of-the-backend-plugin-system

Reply:
Recorded[ a task ](https://github.com/grafana/grafana-plugin-examples/issues/377)to clarify this in backend docs.